docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -309,10 +309,10 @@ ways:
 :Non code content: At the moment, these must also be under the `GPL-3.0-or-later       
                    <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_.
 
-Use `this table of licenses from the Fedora Project
-<https://docs.fedoraproject.org/en-US/legal/allowed-licenses/>`_ to find which licenses are
-compatible with the GPLv3+.  The license must be considered open source on both the Fedora License
-table and the `Debian Free Software Guidelines <https://wiki.debian.org/DFSGLicenses>`_ to be
+Use `the list of licenses from gnu.org
+<https://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses>`_ to find which licenses are
+compatible with the GPLv3+. The license must be considered open source and compatible with GPLv3+ on both the gnu.org license
+list and the `Debian Free Software Guidelines <https://wiki.debian.org/DFSGLicenses>`_ to be
```

Comment:
```suggestion
You can refer to the following resources for license information:
*  `table of licenses from the Fedora Project
<https://docs.fedoraproject.org/en-US/legal/allowed-licenses/>`_ is a good general list.
* `list of licenses from gnu.org
<https://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses>`_ specifies which licenses are
compatible with the GPLv3+.
* the `Debian Free Software Guidelines <https://wiki.debian.org/DFSGLicenses>`_ is another popular list.
The license must be compatible with GPLv3+ and considered open source on both the Fedora and Debian lists to be
```

Reply:
@acozine thanks for the suggestion!
I think this changes the proposal most people have already voted on.
Let's finish with this as is. We can continue working on improving this in following topics / PRs.

Reply:
This looks good to me, but I'd word the last sentence as

> Collections MUST use licenses that are compatible with GPLv3+ and considered open source on both the Fedora and Debian lists.